docs/data-sources/http.md

```diff
@@ -107,5 +107,6 @@ resource "random_uuid" "example" {
 
 - `id` (String) The ID of this resource.
 - `response_body` (String) The response body returned as a string.
+- `response_body_base64_std` (String) The response body encoded as base64 (standard) as defined in [RFC 4648](https://datatracker.ietf.org/doc/html/rfc4648#section-4).
```

Comment:
Ah, I looked at the RFC, and I think what you are tying to convey here is that we are using the standard Base64. But, even in the RFC, it says _<<it might be referred to as "base64">>_

Reply:
I don't think I've ever heard the "standard" part before (which very much intrigued me that there was another shorthand encoding term, "base64url", okay makes sense and today I learned), but take that with whatever grain of salt. 😅

If you're looking for something more concrete with respect to other large Terraform providers, it is safe to bet on calling it `*_base64`. 👍 

Some various references:

* https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_object#content_base64
* https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/key_vault_certificate#certificate_data_base64
* https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/endpoints_service#protoc_output_base64

Reply:
Have removed the `_std` suffix.